doc/teacher_scripts.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Skripte für Lehrende
+---
+
+Für Lehrende stehen neben dem Skript `scripts/deploy_to_grading.py` zur Ausführung der D2G-Pipeline in einem Aufgaben-Repository zwei weitere Skripte zum Einsammeln und Auswerten der Ergebnisse zur Verfügung. Dies sind das Skript `scripts/deploy_to_grading_teacher.py`, welches für ein Aufgabenblatt die Abgaben aller Studierende einsammelt und eine Zusammenfassung der Ergebnisse für dieses Aufgabenblatt ausgibt, und das Skript `scripts/teacher_summary.py`, welches aus den Ergebnissen aller Aufgabenblätter eine Übersicht über die Ergebnisse über z.B. ein gesamtes Semester erstellt.
+
+### deploy_to_grading_teacher.py
+
+Das Skript `scripts/deploy_to_grading_teacher.py` ermittelt für ein Aufgabenblatt die von den Studierenden erreichten Punkte und fasst diese in einer Excel-Datei zusammen. Über den folgenden Befehl kann das Skript gestartet werden. Dabei muss die `D2G_PATH`-Umgebungsvariable gesetzt sein.
```

Comment:
Für mich: 
> `D2G_PATH`-Umgebungsvariable gesetzt sein.

steht das irgendwo, wie das geht? Würde sich hier ein Link zur Erklärung lohnen?

Reply:
Steht in der README im selben Ordner. Habe ich als Hinweis ergänzt.